sys/kern/fdt.c

```diff
@@ -130,14 +144,58 @@ ssize_t FDT_getencprop(phandle_t node, const char *propname, pcell_t *buf,
   if (buflen % 4)
     return -1;
 
-  ssize_t ret = FDT_getprop(node, propname, buf, buflen);
-  if (ret < 0)
+  ssize_t rv = FDT_getprop(node, propname, buf, buflen);
+  if (rv < 0)
```

Comment:
Please avoid name changes if you introduce new features and do that in separate commits.

Reply:
Good point. I removed these changes from this PR.